NetworkingSampleApp/NetworkingSampleApp/API/Routers/SampleUploadRouter.swift

```diff
@@ -23,6 +23,8 @@ enum SampleUploadRouter: Requestable {
             return ["Content-Type": "image/png"]
         case let .file(url):
             return ["Content-Type": url.mimeType]
+        case let .multipart(boundary):
+            return ["Content-Type": "multipart/form-data; boundary=\(boundary)"]
```

Comment:
We needed to drop the first two -- otherwise the (NodeJS) server kept returning "Unexpected end of form". Are there different boundary standards across servers or is this a bug on our side?